lib/config/configItems.joi.js

```diff
@@ -5,6 +5,7 @@ const joi = require('joi');
 const hostPortJoi = joi.object({
     host: joi.string().required(),
     port: joi.number().greater(0).required(),
+    iamPort: joi.number().greater(0).optional(),
```

Comment:
A better place to put this would be by appending `.keys({ iamPort: ...})` to the `hostPortJoi` where it's used, this one shall remain generic.